### PR TITLE
bots: Migrate image-refresh bot to new task structure

### PR DESCRIPTION
**Pull request by stefwalter**
The new image-refresh bot is now driven off of checklists in either issues or pull requests.

Depends on:
 * [x] #6735

### CODE REVIEW

**Comment by petervo** [comment]
Needs rebase

**Comment by stefwalter** [comment]
Tested using #6853 ... but noticed that custom set of triggered images are missing.

**Comment by stefwalter** [comment]
Testing again using #6864

**Comment by stefwalter** [comment]
Successfully did a run with #6867

File: .tasks [hunk 1]
```diff
@@ -13,10 +13,10 @@
 
 set -ex
 bots/tests-scan
-bots/image-scan
 
 # File issues for these tasks
 bots/po-trigger
+bots/image-trigger
```
Comment (martinpitt) [inline .tasks]: This renaming needs to be reflected in bots/README.md

File: bots/tests-trigger [hunk 2]
```diff
@@ -7,18 +7,6 @@ sys.dont_write_bytecode = True
 
 import github
 
-def trigger_image(api, opts):
```
Comment (martinpitt) [inline bots/tests-trigger]: The github-trigger → tests-trigger rename needs to be reflected in bots/README.md.

File: .tasks (hunk 1 above)
Comment (stefwalter) [inline .tasks]: Done.

File: bots/tests-trigger (hunk 2 above)
Comment (stefwalter) [inline bots/tests-trigger]: Done.

**Pull request merged**